helm/odigos/templates/autoscaler/webhooks.yaml

```diff
@@ -48,6 +43,4 @@ metadata:
   annotations:
     "helm.sh/hook": "pre-install,pre-upgrade"
     "helm.sh/hook-delete-policy": "before-hook-creation"
-data:
-  tls.crt: {{ $cert.Cert | b64enc }}
-  tls.key: {{ $cert.Key | b64enc }}
+# the actual certificate will be populated by the controller
```

Comment:
If someone is using helm-controller, then this controller might "fight" over this resource and will attempt to reconcile it back to secret with empty data, no?